daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/Converter.scala

```diff
@@ -771,7 +773,8 @@ object Converter {
       case SVariant(_, "ParticipantAdmin", _, SUnit) =>
         Right(UserRight.ParticipantAdmin)
       case SVariant(_, "IdentityProviderAdmin", _, SUnit) =>
-        Right(UserRight.IdentityProviderAdmin)
+        // TODO(#15857) Add support for the `IdentityProviderAdmin` in the Daml Script
```

Comment:
// TODO https://github.com/digital-asset/daml/issues/15857
// Add support for the `IdentityProviderAdmin` in the Daml Script